PrebidMobile/API1.0Demo/build.gradle

```diff
@@ -31,15 +31,15 @@ android {
     }
 }
 
-project.gradle.taskGraph.whenReady {
-    connectedSourceCodeDebugAndroidTest {
-        ignoreFailures = true
-    }
-}
+//project.gradle.taskGraph.whenReady {
```

Comment:
I have added this setting as a part of #106 

The main reason that DemoApp includes UI tests that rely on Internet connection, real PBS responses etc. As a result we can not guarantee that all tests always are passed. 

We should overview this type of tests.
Please uncomment these lines for now. It should solve TravisCI fail